.github/workflows/on-push.yml

```diff
@@ -59,12 +59,6 @@ jobs:
         if: matrix.os == 'windows-latest'
         run: |
           echo "$HOME\.elan\bin" >> $env:GITHUB_PATH
-      - name: Setup elan toolchain on Windows
```

Comment:
this was a hack since windows bootstrap test was not reliable, but this hack is no longer needed.